lib/screens/v2/location_context.ex

```diff
@@ -9,7 +9,7 @@ defmodule Screens.LocationContext do
   alias Screens.Stops.Stop
   alias Screens.Util
 
-  alias ScreensConfig.V2.{BusEink, BusShelter, Dup, GlEink, PreFare}
+  alias ScreensConfig.V2.{BusEink, BusShelter, Dup, Elevator, GlEink, PreFare}
```

Comment:
The changes to this module aren't necessary anymore now that we aren't using it.